docs/src/hugo/layouts/partials/nav-docs.html

```diff
@@ -1,23 +1,8 @@
-<li class="dropdown">
-  <a class="dropdown-toggle" id="doc-menu-item" data-toggle="dropdown" data-target="#" href="http://http4s.org/" role="button" aria-haspopup="true" aria-expanded="false">
-    Documentation <span class="caret"></span>
-  </a>
-  <ul class="dropdown-menu" aria-labelledby="doc-menu-item">
-    <li><a href="/versions/">Versions</a></li>
-    <!-- v0.17 -->
-    <li role="separator" class="divider"></li>
-    <li class="dropdown-header">v0.17 (development)</li>
-    <li><a href="/v0.17/">Tutorial</a></li>
-    <li><a href="/v0.17/api/org/http4s/">Scaladoc</a></li>
-    <!-- v0.16 -->
-    <li role="separator" class="divider"></li>
-    <li class="dropdown-header">v0.16 (development)</li>
-    <li><a href="/v0.16/">Tutorial</a></li>
-    <li><a href="/v0.16/api/org/http4s/">Scaladoc</a></li>
-    <!-- v0.15 -->
-    <li role="separator" class="divider"></li>
-    <li class="dropdown-header">v0.15</li>
-    <li><a href="/v0.15/">Tutorial</a></li>
-    <li><a href="/v0.15/api/org/http4s">Scaladoc</a></li>
-  </ul>
-</li>
+<div class="dropdown-menu" aria-labelledby="doc-menu-item"> 
+  <a class="dropdown-item" href="/v0.18/">v0.18 (development)</a>
+  <a class="dropdown-item" href="/v0.18/">v0.17 (stable)</a>
+  <a class="dropdown-item" href="/v0.18/">v0.16 (stable)</a>
+  <a class="dropdown-item" href="/versions/">Help me choose...</a>
```

Comment:
This is the only part of the theme I deeply care about going back and republishing old versions when it changes.  Maybe we could stuff a JSON blob out in http4s.org, and have this dropdown populate itself from that.  That could refresh EOLed versions without touching them again.